### PR TITLE
Episode page changes

### PR DESCRIPTION
**Pull request by oliviale**
* Add total episodes count
* Add search UI
* Add sort options
* Change `Play` button to icon button
* Add `Add to queue` button
* Mobile styles

UI only, no feature.

### CODE REVIEW

File: src/ui/EpisodesList/EpisodesList.tsx [hunk 1]
```diff
@@ -21,6 +24,26 @@ export function EpisodesList({ className = '', episodes }: EpisodesListProps) {
 
   return (
     <div className={`${styles.container} ${className}`} ref={containerRef}>
+      {children}
+      <FeatureToggle feature="episodesFilter">
+        <div className={styles.episodeListOptions}>
```
Comment (shantanuraj) [inline src/ui/EpisodesList/EpisodesList.tsx:29]: The episodes filter also gave me an idea to fix the virtual list by putting the content by rendering the episode info inside the list item component as the scroll seems to have been fixed.

File: src/ui/EpisodesList/EpisodesList.tsx [hunk 2]
```diff
@@ -21,6 +24,26 @@ export function EpisodesList({ className = '', episodes }: EpisodesListProps) {
 
   return (
     <div className={`${styles.container} ${className}`} ref={containerRef}>
+      {children}
+      <FeatureToggle feature="episodesFilter">
```
Comment (shantanuraj) [inline src/ui/EpisodesList/EpisodesList.tsx:28]: Guarding this behind a flag so we can turn it on separetly from this PR, I'll do the implementation in a follow-up.
To view it locally edit the `src/shared/features/Features.tsx` file and set `episodesFilter` to `true`

File: src/ui/EpisodesList/EpisodeItem.module.css [hunk 3]
```diff
@@ -1,43 +1,68 @@
 .container {
   display: flex;
   align-items: center;
-  flex-direction: row;
-  padding: 0 32px;
+  column-gap: 15px;
```
Comment (shantanuraj) [inline src/ui/EpisodesList/EpisodeItem.module.css:4]: Didn't know `column-gap` works with `flex`.

File: src/shared/toast/Toast.tsx [hunk 4]
```diff
@@ -0,0 +1,16 @@
+import { useEffect } from 'react';
+
+import styles from './Toast.module.css';
+import { getClearToast, getMessage, toastTimeout, useToast } from './useToast';
+
+export const Toast = () => {
```
Comment (shantanuraj) [inline src/shared/toast/Toast.tsx:6]: Threw together a toast component, it doesn't look too fancy atm.

**Review by shantanuraj — APPROVED** [review]
LGTM 🚀 

You can try the preview at
https://podcst-web-8zhohmj02-shantanu.vercel.app/feed/top

Feel free to merge after checking it out.
I'll implement the episodes filter functionality in a follow up PR.

File: src/ui/EpisodesList/EpisodeItem.module.css (hunk 3 above)
Comment (oliviale) [reply src/ui/EpisodesList/EpisodeItem.module.css:4]: it's a new thing that is finally available in all modern browsers :D

**Comment by oliviale** [comment]
Mm special effect in mobile where the items disappear on scroll, or looks like it does.

https://user-images.githubusercontent.com/9974754/135739689-e3a81164-4cdb-45b6-af40-1b8a581b6d2d.mov

**Comment by shantanuraj** [comment]
Strangely it seems to be miscalculating the top offset only on mobile by 2 items. (200px)
looking into it

**Comment by shantanuraj** [comment]
Band-aided by using a different layout on mobile, lmk if it's alright

https://user-images.githubusercontent.com/1716853/135747044-f70c89a6-509a-44f9-8e05-cfd3e45eaeea.mp4

**Pull request merged**